src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String draw() {
+        String a = "Figure: " + this.getClass().getSimpleName() + ", area: " + getArea();
+        String b = " sq.units, radius: " + radius + " units, color: " + getRandomColor();
+        return a + b;
```

Comment:
```suggestion
        return "Figure: " + this.getClass().getSimpleName() + ", area: " + getArea();
        + " sq.units, radius: " + radius + " units, color: " + getRandomColor();
```
a and b are not informative, it's better to return a string, or give it some meaningful name